src/tests/AttributesManagersTest.cpp

```diff
@@ -16,6 +16,7 @@
 
 namespace Cr = Corrade;
 
+using namespace Magnum::Math::Literals;
```

Comment:
I think clang tidy will complain on this, there's some suppression that you have to use I think (search for `Literals` in the codebase)